### DISCUSSION TITLE
Sesi 10-NextJS

### DISCUSSION THREAD

**Discussion by CentaurId** [deaafrizal/forum#30] [Troubleshooting Room]
WTA abang-abang semua, ada kendala error buat addToCollection nih bang. Ada yang tau sebab sama gimana fix-inya ngga ya bang ?
![collectionButton](https://github.com/deaaprizal/forum/assets/119429252/13935297-bff6-45d5-b1d2-236e9a2d011c)
<img width="954" alt="hasilCollectionButton" src="https://github.com/deaaprizal/forum/assets/119429252/4750f50d-91d7-4353-9f9d-1da56ddbf796">

**Comment by deaafrizal** [comment]
data yang dikirim ke api kayanya ada salah parameter / key, coba ulik dibagian fetchingnya aja. make sure data depan & belakang sesuai

**Reply by CentaurId** [reply]
parama/key yang bagian mana ya bang ? soalnya sudah sesuai semua sepertinya

<img width="751" alt="param1" src="https://github.com/deaaprizal/forum/assets/119429252/082e6e0b-9f0e-47e4-9569-0a5dab1d820f">
<img width="752" alt="param2" src="https://github.com/deaaprizal/forum/assets/119429252/c912faff-5ae4-4e5a-b095-ca8ab45b3861">
<img width="612" alt="param3" src="https://github.com/deaaprizal/forum/assets/119429252/b2335a98-e12b-47e2-8d0c-06fdac9f449a">

**Comment by danielpsrb** [comment]
pastikan nama tabel yang kamu buat didatabase untuk collection sesuai dengan yang di route.js ,, 

const createCollection = await prisma.collection.create({ data })

kalau gak sama, error jadi nya, jdi hrus disesuaikan
